ext/nnstreamer/tensor_decoder/tensordec-directvideo.c

```diff
@@ -88,7 +153,24 @@ dv_getOutCaps (void **pdata, const GstTensorsConfig * config)
       format = GST_VIDEO_FORMAT_RGB;
       break;
     case 4:
-      format = GST_VIDEO_FORMAT_BGRx;
+      if (ddata->format == DIRECT_VIDEO_FORMAT_RGBx)
```

Comment:
better changing switch (ddata->format)..case

Reply:
it's applied